event_registration_cancel_reason/models/event_registration.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L.
+# © 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    cancel_reason_id = fields.Many2one(
+        comodel_name='event.registration.cancel.reason', readonly=True,
+        string="Cancellation reason", ondelete="restrict")
```

Comment:
Be consistent: everywhere it should be "Cancellation reasons" (instead of Cancel Reason)